markdown/events/2022/bytesize-40-software-packaging.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Bytesize 40: Software packaging"
+subtitle: Alexander Peltzer - Boehringer Ingelheim Pharma GmbH & Co. KG, Germany
+type: talk
+start_date: "2022-04-05"
+start_time: "13:00 CET"
+end_date: "2022-03-01"
```

Comment:
```suggestion
end_date: "2022-04-05"
```

Reply:
I have a feeling that the `end_date` can actually just be omitted if it's a single day event..? But my memory is a bit rusty and I guess it doesn't hurt..